raven/handlers/logging.py

```diff
@@ -174,7 +174,14 @@ def _emit(self, record, **kwargs):
         tags.update(getattr(record, 'tags', {}))
 
         kwargs.update(handler_kwargs)
+        sample_rate = extra.pop('sample_rate', None)
```

Comment:
i think pop is good because theres no real reason to pass this through contextually if its thought of as configuration